### DISCUSSION TITLE
How does the plugin access local models such as bge-small-en-v1.5

### DISCUSSION THREAD

**Discussion by ghost** [brianpetro/obsidian-smart-connections#833] [General]
I selected Xenove/bge-small-en-v1.5 and its working, but i don'T understand how. I never installed this model on my computer. Does the Smart Connections plugin download it automatically?

Thanks

**Comment by brianpetro** [comment]
The local embedding models are handled by transformers.js, a huggingface project, which downloads the model on-demand from huggingface servers 🌴

**Reply by ghost** [reply]
thanks for clearing that up
